### PR TITLE
Version 0.4.31

### PR DESCRIPTION
**Pull request by tshemsedinov**

### CODE REVIEW

File: deploy/centos7x64-mongo.sh [hunk 1]
```diff
@@ -10,4 +10,4 @@ sudo yum install -y mongodb-org
 service mongod start
 chkconfig mongod on
 cd /ias
-sudo npm install mongodb@2
```
Comment (nechaido) [inline deploy/centos7x64-mongo.sh]: We should use specific version here.

File: deploy/debian-mongo.sh [hunk 2]
```diff
@@ -6,4 +6,4 @@ sudo apt-get install -y mongodb-org
 sudo service mongod start
 sudo update-rc.d mongod defaults
 cd /ias
-sudo npm install mongodb@2
+sudo npm install mongodb
```
Comment (nechaido) [inline deploy/debian-mongo.sh]: Ditto.

File: deploy/fedora-mongo.sh [hunk 3]
```diff
@@ -3,4 +3,4 @@ sudo dnf install -y mongodb-server
 sudo systemctl enable mongod
 sudo systemctl start mongod
 cd /ias
-sudo npm install mongodb@2
+sudo npm install mongodb
```
Comment (nechaido) [inline deploy/fedora-mongo.sh]: Ditto.

File: deploy/ubuntu-mongo.sh [hunk 4]
```diff
@@ -11,4 +11,4 @@ sudo apt-get -y install mongodb-org
 sudo service mongod start
 sudo update-rc.d mongod defaults
 cd /ias
-sudo npm install mongodb@2
+sudo npm install mongodb
```
Comment (nechaido) [inline deploy/ubuntu-mongo.sh]: Ditto.

File: deploy/centos7x64.sh [hunk 5]
```diff
@@ -3,7 +3,7 @@ yum -y update
 yum -y install wget mc
 yum -y install psmisc
 yum -y groupinstall "Development Tools"
-curl --silent --location https://rpm.nodesource.com/setup_9.x | bash -
+curl --silent --location https://rpm.nodesource.com/setup_7.x | bash -
```
Comment (nechaido) [inline deploy/centos7x64.sh]: What is the purpose of this downgrade?.

File: deploy/debian.sh [hunk 6]
```diff
@@ -4,7 +4,7 @@ sudo apt-get -y update
 sudo apt-get -y install wget mc curl
 sudo apt-get -y install build-essential openssl libssl-dev pkg-config
 sudo apt-get -y install python
-curl -sL https://deb.nodesource.com/setup_9.x | sudo -E bash -
+curl -sL https://deb.nodesource.com/setup_7.x | sudo -E bash -
```
Comment (nechaido) [inline deploy/debian.sh]: Ditto.

File: deploy/fedora.sh [hunk 7]
```diff
@@ -1,9 +1,9 @@
 #!/bin/bash
 dnf -y update
-dnf -y install wget mc gcc-c++
+dnf -y install wget mc
 dnf -y install psmisc
 dnf -y groupinstall "Development Tools"
-curl --silent --location https://rpm.nodesource.com/setup_9.x | bash -
+curl --silent --location https://rpm.nodesource.com/setup_7.x | bash -
```
Comment (nechaido) [inline deploy/fedora.sh]: Ditto.

File: deploy/ubuntu.sh [hunk 8]
```diff
@@ -3,7 +3,7 @@ RELEASE=$(lsb_release -sc)
 sudo apt-get -y update
 sudo apt-get -y install wget mc curl
 sudo apt-get -y install build-essential openssl libssl-dev pkg-config python
-curl -sL https://deb.nodesource.com/setup_9.x | sudo -E bash -
+curl -sL https://deb.nodesource.com/setup_7.x | sudo -E bash -
```
Comment (nechaido) [inline deploy/ubuntu.sh]: Ditto.

File: deploy/fedora.sh [hunk 9]
```diff
@@ -1,9 +1,9 @@
 #!/bin/bash
 dnf -y update
-dnf -y install wget mc gcc-c++
```
Comment (nechaido) [inline deploy/fedora.sh]: `gcc-c++` must be installed.

**Comment by tshemsedinov** [comment]
Sorry, I don't know how could I pushed such a...

**Comment by tshemsedinov** [comment]
Landed in a20fb31

**Pull request closed without merging**